README.MD

```diff
@@ -0,0 +1,28 @@
+# AC-Configbuilder
+_Beschreibung des Projekts_
+
+## Struktur
+_hier muss die spätere Strucktur erklärt werden_
+
+
+### Was soll es können
+**1.  .config Datei erstellen, die AC-Datein n-fach erstellen kann.**
+   * Config Strucktur erstellen [GUID](https://www.audiocodes.com/media/13244/gateway-and-sbc-cli-reference-guide-ver-72.pdf)
+   * .json file für Config
+   * Konfigurationen als Template definieren
+
+**2. ID's über API ermitteln und einfügen**
+  * API ansprechen [GUID](https://www.audiocodes.com/media/13528/rest-api-for-mediant-devices-ver-72.pdf)
+  * festgelegte ID's abrufen und übernehmen
+ 
+**3. Konfigurationen automatisch einspielen**
+  * n-viele Konfiguratioenn einspielen [GUID](https://www.audiocodes.com/media/13528/rest-api-for-mediant-devices-ver-72.pdf)
+  * multible Devices zulassen
+
+### Funktionen
+Das Tool ist momentan über 2 Kommands in Powershell steuerbar und kann benutzerdefinierte einge wie den Path zu einer Datei bearbeiten und auswerten. 
```

Comment:
Dieser Satz hier ist irgendwie mist. Und auch von der Herangehensweise nich so richtig gut. Du kannst ja mal in der PowerShell ```Get-Help```eingeben. Das ist wohl eher die Art von Text die hier hin muss.

Reply:
Ich werde das anpassen wenn das Tool auf Ver.1 funktioniert.
Bis dahin lass ich das mal aus.